fmutils.go

```diff
@@ -150,3 +155,31 @@ func (mask NestedMask) Prune(msg proto.Message) {
 		return true
 	})
 }
+
+// Override - overrides all of the fields listed in paths in dest msg using the values from the src msg.
```

Comment:
nit: better comment message

Reply:
Is this a TODO? If so, please address it.

Reply:
Addressed